pkg/graph/schema.resolvers.go

```diff
@@ -232,6 +236,154 @@ func (r *queryResolver) Systems(ctx context.Context, after *string, first int) (
 	return conn, nil
 }
 
+func (r *systemIntakeResolver) BusinessNeed(ctx context.Context, obj *models.SystemIntake) (*string, error) {
```

Comment:
These methods get generated when there isn't a matching field on the struct type (`models.SystemIntake` in this case). I want to say that some of these we don't need since they are just accessing a struct field, but we might need to create a different struct if there is a big mismatch between how we are exposing the data in the graph and how it is laid out in the storage layer.